include/ck/tensor_operation/gpu/block/thread_group_tensor_slice_transfer_v7.hpp

```diff
@@ -0,0 +1,161 @@
+#pragma once
+
+#include "common_header.hpp"
+#include "tensor_descriptor.hpp"
+#include "tensor_descriptor_helper.hpp"
+#include "cluster_descriptor.hpp"
+#include "threadwise_tensor_slice_transfer_v7.hpp"
+
+namespace ck {
+
+// this version does following things to avoid scratch memory issue
+// 1. Use StaticallyIndexedArray instead of C array for thread buffer
+// 2. ThreadwiseTensorSliceTransfer_v3 does not keep reference to tensor descriptor
```

Comment:
Could you update comments to reflect what's different in `v7`?

Reply:
Done